src/class/hash.rs

```diff
@@ -6,7 +6,7 @@ use types::{Value, ValueType};
 use {AnyObject, Object, VerifiedObject};
 
 /// `Hash`
-#[derive(Debug, PartialEq)]
+#[derive(Copy, Debug, PartialEq)]
```

Comment:
:ok_hand: I only added `Copy` because the `Clone` implementation was implemented via `_dup`, and I figured that was OK.